src/main/java/org/takes/tk/TkFailure.java

```diff
@@ -81,8 +81,8 @@ public TkFailure(final Scalar<IOException> err) {
         super(
             new Take() {
                 @Override
-                public Response act(final Request request) throws IOException {
-                    throw err.get();
+                public Response act(final Request request) throws Exception {
```

Comment:
@andreoss Why throw `Exception` here instead of the original `IOException`?

Reply:
@paulodamaso `Take` & `Scalar` both throw `Exception`. 
I've changed it back to `IOException.